_pages/en/it-picture-long-term.md

```diff
@@ -28,8 +32,11 @@ These citizens receive their benefits seamlessly throughout their lives as major
 The GC (OneGC) Portal provides access to services and benefits (not just ESDC) in tell us once fashion.
 Registering to the Portal will create or use the trusted Canadian profile and enable automatic enrollment and delivery in the future.
 Help for using the Portal is available directly on the Portal using instant messaging (text, voice or video) as well as by phone.
+This assistance will be delivered by AI or services center staff.
```

Comment:
Tangible? Citizen shouldn't care if it's delivered through AI or Assembly. 

Perhaps it... 

- Results in easier access to services through delivering services to the client through hands-off automated processes or supplemented by staff if required

Reply:
We we're trying to add AI in there..  I'll add your suggestions